### ISSUE TITLE
Header View

### ISSUE THREAD

**Issue by ueen** [MinSeungHyun/slush#24]
If i might suggested another feature:
A different header view that is displayed as the first item would be awesome could be something like this in the builder:
`addHeader(R.layout.header) { view -> ... }`

And also a footer as the last item should be added.

I think then this library really suits all basic needs one can wish for 😎

**Comment by ueen** [comment]
Here are several ways to do it https://khacpv.medium.com/android-best-practice-for-recyclerview-with-headerview-140bd4b10757
And another https://medium.com/@polson55/the-best-way-to-add-a-recyclerview-header-f29f3ff56f84

Ok I think the last one is not that good because there are problems if it gets nested, actually I think the easiest would be to handle multiple viewholder types and calculate offset, for that slush has to support multiple types (which is on the TODO) or easier would probably be to just use ConcatAdapter as described here
https://medium.com/androiddevelopers/get-ahead-using-headers-in-recyclerview-2909a69b19

@MinSeungHyun you know slush best, what do you think is the best/easiest route, add multiple holdertype support first or concat adapters?

**Comment by MinSeungHyun** [comment]
Thank you so much for helping my library 🥰
But I'm very busy these days because I have an important exam coming up at the end of this year. I'll make sure to check it out after the exam!!

**Comment by ueen** [comment]
You're welcome, no problem! 
Let me know if you want me to work on something and do a PR 😇

**Comment by ueen** [comment]
Hope you did well on your exam! Any progress so far? 😊

**Comment by MinSeungHyun** [comment]
Yeah, thank you for your concerns. Unfortunately, as you can see on the description of this repository, I decided not to update this library TT
I'm sorry and thank you again!

**Comment by ueen** [comment]
Oh very sad to hear, you should really reconsider, this is the best/easiest library to handle recyclerviews! ❤

**Comment by MinSeungHyun** [comment]
But there is a way to maintain this library.
You can fork this repository and update it!
